src/de/espend/idea/shopware/util/ShopwareUtil.java

```diff
@@ -53,47 +53,247 @@
         AND TABLE_NAME LIKE '%attributes'
      */
     final public static String[] MODEL_STATIC_ATTRIBUTES = new String[] {
```

Comment:
I don't know if this const should contain all tables. I don't know the code, but from the name, if would guess it contains only the attribute tables